README.md

```diff
@@ -9,8 +9,8 @@ Example usage
 To initialise your environment in utop:
 ```
 open Lwt;;
-#require "vhd-format";;
-#require "vhd-format.lwt";;
+#require "vhd";;
```

Comment:
Do we need ti update these as well?

Reply:
Yes, good spot, I'll update them. However, the example seems to have got out of sync with the source code.